.github/workflows/intel-macs.yml

```diff
@@ -0,0 +1,178 @@
+name: Validate Intel Macs
+
+# This workflow will send notifications only to me (geekosaur) if it fails.
```

Comment:
It's a low bus factor. Any change it could do something more observable? Send an email to cabal-devs? Send a message to the Matrix channel?

Reply:
See https://github.com/haskell/cabal/pull/9437#discussion_r1393937515: this is a GitHub limitation. The notification is sent to the last person who committed a change to a `- cron:` line. If you want something else, you'll need to submit a feature request against GitHub.

Reply:
I don't think there's a webhook for scheduled workflows, but I can check.

Reply:
Turns out all workflows can trigger webhooks at various points; I would have to enable the webhook for `workflow_run` and intercept the `completed` event.

(But that just leads us to the meta version: unless someone e.g. HF takes over the Matrix bot, which is currently part of the stack including the bridge, the bus factor is still 1 and may be even lower depending on political events in the near future here.)